versions.yaml

```diff
@@ -159,10 +159,9 @@ assets:
       url: "https://github.com/intel/linux-kernel-dcp/archive/refs/tags"
       tag: "SPR-BKC-PC-v9.6"
     sev:
-      description: "Linux kernel with efi_secret support"
-      url: "https://github.com/confidential-containers-demo/\
-      linux/archive/refs/tags/"
-      tag: "efi-secret-v5.17-rc6"
+      description: "Linux kernel with SEV support"
+      url: "https://cdn.kernel.org/pub/linux/kernel/v5.x/"
+      tag: "v5.19"
```

Comment:
Also update this one to v5.19.1

Reply:
I think it should be `version:` instead of `tag:`, because you're downloading from the CDN (and not from github).

Reply:
Version updated. @dubek you mind if we address the version/tag item in a separate issue?